### PR TITLE
Adding single field index for cosmos db

### PR DESCRIPTION
**Pull request by pradeepvairamani**
Azure fluid relay uses [cosmos db api for mongo db](https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb/mongodb-introduction) for storing the ops. In cosmos db:
1) Unique and compound indexes do not support nested paths
2) Compound indexes are only used in queries that sort results and not for filtering

src: https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb/mongodb-indexing

This was resulting in the createIndex calls failing silently and this resulted in high latency for db reads. This PR changes the compound indexes to multiple single field indexes as suggested by the cosmos team. 

Verification: We already verified that by manually changing to single field indexes, we were seeing a significant improvement in the getDeltas call latency and the session boot time.

Screen Shot 2021-10-30 at 8 28 08 PM

This also improved the duration of the e2e tests significantly.

Also, verified the code changes in a dev cluster to make sure that the indexes are getting created successfully.

### CODE REVIEW

File: server/routerlicious/packages/routerlicious/src/scribe/index.ts [hunk 1]
```diff
@@ -46,16 +46,18 @@ export async function scribeCreate(config: Provider): Promise<IPartitionLambdaFa
...
-        {
-            "documentId": 1,
-            "operation.sequenceNumber": 1,
-            "tenantId": 1,
-        },
-        true);
-
     if (createCosmosDBIndexes) {
+        await scribeDeltas.createIndex({ documentId: 1 }, false);
+        await scribeDeltas.createIndex({ tenantId: 1 }, false);
```
Comment (hedasilv) [inline server/routerlicious/packages/routerlicious/src/scribe/index.ts:51]: Is there any implication or unexpected side effect now that we have to set `unique` to `false`?

**Review by hedasilv — APPROVED** [review]

File: server/routerlicious/packages/routerlicious/src/scribe/index.ts (hunk 1 above)
Comment (pradeepvairamani) [reply server/routerlicious/packages/routerlicious/src/scribe/index.ts:51]: Actually, this is something I was thinking too. Now that we have document id as a guid, we could make the docid uniqueness set to true. @tanviraumi thoughts?

Comment (tanviraumi) [reply server/routerlicious/packages/routerlicious/src/scribe/index.ts:51]: "Unique" is probably per entry level? So each document will have multiple entries (one per operation) with the same documentId.

File: server/routerlicious/packages/routerlicious/src/scriptorium/index.ts [hunk 2]
```diff
@@ -26,19 +26,18 @@ export async function create(config: Provider): Promise<IPartitionLambdaFactory>
...
-        true);
 
     if (createCosmosDBIndexes) {
+        await opCollection.createIndex({
+            documentId: 1,
+        }, false);
+
+        await opCollection.createIndex({
+            tenantId: 1,
+        }, false);
```
Comment (tanviraumi) [inline server/routerlicious/packages/routerlicious/src/scriptorium/index.ts:37]: nit: Make each one single line like the scribe/index file.

**Review by tanviraumi — APPROVED** [review]

**Pull request closed without merging**